doc/source/raysgd/v2/user_guide.rst

```diff
@@ -126,42 +128,72 @@ Porting code to RaySGD
 
         TODO. Write about how to convert code to use horovod.
 
+.. _sgd-logging:
 
+Logging
```

Comment:
Can we add a section for the different backends here?

Reply:
Also we should have advanced usage section with the `run_iterator` and Class APIs

Reply:
1. Added a `Backends` section to summarize the supported backends.
2. Added a commented-out section for the advanced APIs. Will populate this in more detail in a separate PR.
3. Will follow-up in a separate PR to populate the `Porting code to RaySGD` section.